packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -115,6 +119,17 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
     [iframeCoordinatesToWindowCoordinates, onMoveComponent],
   );
 
+  const [bodyFocused, setBodyFocused] = React.useState(false);
+  const handleFocus = (ev: FocusEvent) => {
+    if (ev.target && (ev.target as any).getAttribute('data-builder-id') === null) {
```

Comment:
nit: consider putting the expression `ev.target && (ev.target as any).getAttribute('data-builder-id') === null` in a well named variable and using `bodyFocused` in the if clause. I think it might be easier to read.

Certainly took me a while to get it :)